### PR TITLE
Update for _0x79C0E43EB9B944E2

### PR DESCRIPTION
**Pull request by azutake**
I found this while implementing the lucky wheel.
It seems to be native to change the camera view for some table games.

I used the name as I found it from [GTA5 NativeDB](https://alloc8or.re/gta5/nativedb/).

### CODE REVIEW

File: CAM/SetTableGamesCameraThisUpdate.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+```c
+enum eTableGames {
+  LuckeyWheel = 5891389,
+  Slots = 518572876,
+  Roulette = 71681063,
+  Blackjack = -2124244681,
+  ThreeCard = -1938411241,
+}
+```
```
Comment (PsychoShock) [inline CAM/SetTableGamesCameraThisUpdate.md:22]: Change this for a table method.

File: CAM/SetTableGamesCameraThisUpdate.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+## Parameters
+* **hash**:
```
Comment (PsychoShock) [inline CAM/SetTableGamesCameraThisUpdate.md:25]: Missing param description

File: CAM/SetTableGamesCameraThisUpdate.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+## Return value
+
+Returns camera successfully switched.
+
```
Comment (PsychoShock) [inline CAM/SetTableGamesCameraThisUpdate.md:30]: Remove space here

File: CAM/SetTableGamesCameraThisUpdate.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+aliases: ["_0x79C0E43EB9B944E2"]
+---
+## SET_TABLE_GAMES_CAMERA_THIS_UPDATE
+
+```c
+// 0x79C0E43EB9B944E2
+BOOL SET_TABLE_GAMES_CAMERA_THIS_UPDATE(Hash hash);
+```
+
+Switch to table game camerawork for the current frame only.
```
Comment (PsychoShock) [inline CAM/SetTableGamesCameraThisUpdate.md:12]: It doesn't "switch" it overrides the third person camera.

File: CAM/SetTableGamesCameraThisUpdate.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+## Return value
+
+Returns camera successfully switched.
+
+## Examples
+
+```lua
+CreateThread(function()
+    while true do
+      SetTableGamesCameraThisUpdate(5891389)
```
Comment (PsychoShock) [inline CAM/SetTableGamesCameraThisUpdate.md:36]: Add a comment that `5891389` is the lucky wheel.

**Review by PsychoShock — CHANGES_REQUESTED** [review]

**Comment by azutake** [comment]
@PsychoShock
Thanks for the review!
I fixed it! How does it look like this?

File: CAM/SetTableGamesCameraThisUpdate.md [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+
+```c
+// 0x79C0E43EB9B944E2
+BOOL SET_TABLE_GAMES_CAMERA_THIS_UPDATE(Hash hash);
+```
+
+Override to table game camerawork for current frame only.
+
+```lua
+eTableGames = {
```
Comment (AvarianKnight) [inline CAM/SetTableGamesCameraThisUpdate.md:15]: enums are typically done in c/cpp fashion

```c
enum eTableGames {
  LuckyWheel = 5891389,
  Slots = 518572876,
  Roulette = 71681063,
  Blackjack = -2124244681,
  ThreeCard = -1938411241,
}
```

File: CAM/SetTableGamesCameraThisUpdate.md [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+  LuckeyWheel = 5891389,
```
Comment (AvarianKnight) [inline CAM/SetTableGamesCameraThisUpdate.md:16]: Luckey -> Lucky

File: CAM/SetTableGamesCameraThisUpdate.md [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+aliases: ["_0x79C0E43EB9B944E2"]
+---
+## SET_TABLE_GAMES_CAMERA_THIS_UPDATE
+
+```c
+// 0x79C0E43EB9B944E2
+BOOL SET_TABLE_GAMES_CAMERA_THIS_UPDATE(Hash hash);
+```
+
+Override to table game camerawork for current frame only.
```
Comment (PsychoShock) [inline CAM/SetTableGamesCameraThisUpdate.md:12]: Add `third person camera`.

File: CAM/SetTableGamesCameraThisUpdate.md [hunk 9]
```diff
@@ -0,0 +1,40 @@
...
+  LuckeyWheel = 5891389,
+  Slots = 518572876,
+  Roulette = 71681063,
+  Blackjack = -2124244681,
+  ThreeCard = -1938411241,
+}
+```
+
+## Parameters
+* **hash**: Hash for table games
```
Comment (PsychoShock) [inline CAM/SetTableGamesCameraThisUpdate.md:25]: `game`* without `s`. It's __one__ game per hash.

File: CAM/SetTableGamesCameraThisUpdate.md [hunk 10]
```diff
@@ -0,0 +1,40 @@
+---
+ns: CAM
+aliases: ["_0x79C0E43EB9B944E2"]
```
Comment (AvarianKnight) [inline CAM/SetTableGamesCameraThisUpdate.md:3]: Remove the underscore in front of the native hash

**Review by AvarianKnight — CHANGES_REQUESTED** [review]

File: CAM/SetTableGamesCameraThisUpdate.md (hunk 6 above)
Comment (azutake) [reply CAM/SetTableGamesCameraThisUpdate.md:15]: @AvarianKnight I had written that, but fix it when the reviewer pointed it out.

File: CAM/SetTableGamesCameraThisUpdate.md (hunk 7 above)
Comment (azutake) [reply CAM/SetTableGamesCameraThisUpdate.md]: Fixed!

File: CAM/SetTableGamesCameraThisUpdate.md (hunk 10 above)
Comment (azutake) [reply CAM/SetTableGamesCameraThisUpdate.md]: Fixed!

**Comment by azutake** [comment]
@AvarianKnight Thanks for the review! I fixed it

File: CAM/SetTableGamesCameraThisUpdate.md [hunk 11]
```diff
@@ -0,0 +1,40 @@
...
+
+```lua
+eTableGames = {
+  LuckeyWheel = 5891389,
+  Slots = 518572876,
+  Roulette = 71681063,
+  Blackjack = -2124244681,
+  ThreeCard = -1938411241,
+}
+```
```
Comment (PsychoShock) [inline CAM/SetTableGamesCameraThisUpdate.md]: Table method is: https://github.com/citizenfx/natives/commit/12f272480700b1a65b0b4be45caf3d506f4a8aa9#diff-fe04c88c56a62288febcdf5b621e3aed74624d92f1434aecc868e7aed0ee8864R17-R25

So do:

```
|   Hash        |    Game         |
| :-----------:| :-------------:  |
|   `5891389`   |  Lucky Wheel    |
|  `518572876`  |     Slots       |
|   `71681063`  |    Roulette     |
| `-2124244681` |    Blackjack    |
| `-1938411241` |   Three Cards   |
```

File: CAM/SetTableGamesCameraThisUpdate.md [hunk 12]
```diff
@@ -0,0 +1,40 @@
...
+
+## Parameters
+* **hash**: Hash for table games
+
+## Return value
+Returns camera successfully overrided.
```
Comment (PsychoShock) [inline CAM/SetTableGamesCameraThisUpdate.md:28]: Returns **_if_** camera **_is_** successfully overrided.

**Review by PsychoShock — CHANGES_REQUESTED** [review]

File: CAM/SetTableGamesCameraThisUpdate.md (hunk 11 above)
Comment (azutake) [reply CAM/SetTableGamesCameraThisUpdate.md]: I see! I didn't understand what you meant by table method, you mean write it in markdown.

**Comment by azutake** [comment]
@PsychoShock Thanks for pointing that out! Fixed!

**Review by PsychoShock — APPROVED** [review]
Make sure to [squash it](https://docs.fivem.net/docs/contributing/git/squash-guide/).

**Comment by azutake** [comment]
OK, I squashed

**Comment by ahcenezdh** [comment]
You can complete the enum TABLE_GAMES_CAMERA_TYPE, check [here](https://vhub.wiki/enums/TABLE_GAMES_CAMERA_TYPE)

**Comment by azutake** [comment]
I was shown the page and found the following information from speculation.
Do you want to commit this information to the document?

|                  HashKey                    |       Hash        |    Game         |
| :---------------------------------: | :-----------:| :-------------:  |
| `CASINO_LUCKY_WHEEL_CAMERA` |   `5891389`   |  Lucky Wheel    |
| `CASINO_SLOT_MACHINE_CAMERA` |  `518572876`  |     Slots       |
| `CASINO_ROULETTE_CAMERA` |   `71681063`  |    Roulette     |
| `CASINO_BLACKJACK_CAMERA` | `-2124244681` |    Blackjack    |
| `CASINO_POKER_CAMERA` | `-1938411241` |   Three Cards   |
| `CASINO_INSIDE_TRACK_CAMERA` | `1929822423` |   Inside Track    |
| `?` | `545868034` |   LoveProfessorP1   |
| `?` | `935304251` |   LoveProfessorP2   |

**Comment by azutake** [comment]
As for LoveProfessor, a hint might help me find the hash key. where is LoveProfessor used?

**Comment by PsychoShock** [comment]
It's `ARCADE_LOVE_PROFESSOR_P1_CAMERA` and `ARCADE_LOVE_PROFESSOR_P2_CAMERA`.

**Comment by azutake** [comment]
@PsychoShock Amazing!! Thank you! Now the unknown is gone!

**Review by 4mmonium — APPROVED** [review]
LGTM, merging. Thank you for your contribution! 😊

**Pull request merged**